portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -57,14 +57,18 @@
   private final static String MESSAGE_PARAMETER = "text-input";
   private final static String SENDER_PARAMETER = "sender";
   private final static String TAG_PARAMETER = "tags";
+
+  // TODO: ADD TO FORM IN INDEX
+  private final static long PARENT_ID_PARAMETER = 0;
   
-  private void putBlogsInDatastore(String tag, String message, String nickname, List<String> reply) {
+  private void putBlogsInDatastore(String tag, String message, String nickname, List<BlogMessage> reply, long parentID) {
     Entity blogMessageEntity = new Entity("blogMessage");
     blogMessageEntity.setProperty("nickname", nickname);
     blogMessageEntity.setProperty("text", message);
     blogMessageEntity.setProperty("time", System.currentTimeMillis());
     blogMessageEntity.setProperty("tag", tag);
     blogMessageEntity.setProperty("replies", reply);
+    blogMessageEntity.setProperty("parentID", parentID);
```

Comment:
Nit: strictly speaking, I think all the properties string (line 65-71) should be defined as constant variables, but I am fine with changing that in some dedicated PRs to clean things up given we already use it for other properties.